views/js/pciCreator/ims/mathEntryInteraction/creator/widget/states/Map.js

```diff
@@ -178,7 +178,7 @@ define([
     // forming gap response object to be further processed by the latexGapInput event
     MathEntryInteractionStateResponse.prototype.getGapResponseObject = function getGapResponseObject(response) {
         return {
-            list: {
+            base: {
```

Comment:
I know the feature wasn't working in the last versions of the PCI but it looks like changing the response cardinality will introduce a breaking change. From the response processing and scoring, I think we should keep the response as an array of string `["gap1","gap2"]`, ie `RESPONSE: { list: ["gap1","gap2"] } }`

Reply:
changes reverted here